ccd-definition/FixedLists/CMONextHearingStatus.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CMONextHearingStatus",
```

Comment:
Same here, I'd use word **type** to reflect it is type. I am also not sure whether **next** needs to be there.